.gitignore

```
@@ -95,11 +95,7 @@ log/*
 pyvenv.cfg
 
 # STM32
-.metadata
-.settings
-Release\
-Debug\
-*.launch
+.metadata/
```

```suggestion
.metadata\
.settings
Release\
Debug\
*.launch
```
These are needed, right?